packages/ui/src/forum/components/Thread/ThreadTitle.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useCallback, useState } from 'react'
+import styled from 'styled-components'
+import * as Yup from 'yup'
+
+import { InputComponent, InputText } from '@/common/components/forms'
+import { CrossIcon, VerifiedMemberIcon } from '@/common/components/icons'
+import { EditSymbol } from '@/common/components/icons/symbols'
+import { PageTitle } from '@/common/components/page/PageTitle'
+import { useForm } from '@/common/hooks/useForm'
+import { useModal } from '@/common/hooks/useModal'
+import { EditThreadTitleModalCall } from '@/forum/modals/EditThreadTitleModal'
+import { ForumThreadWithDetails } from '@/forum/types'
+import { useMyMemberships } from '@/memberships/hooks/useMyMemberships'
+
+interface ThreadTitleProps {
+  thread: ForumThreadWithDetails
+}
+
+interface TitleFormFields {
+  title: string
+}
+
+const FormSchema = Yup.object().shape({})
+
+export const ThreadTitle = ({ thread }: ThreadTitleProps) => {
+  const { members: myMembers } = useMyMemberships()
+  const [isEditTitle, setEditTitle] = useState<boolean>(false)
+  const { showModal, hideModal } = useModal<EditThreadTitleModalCall>()
+
+  const isMyThread = thread && myMembers.find((member) => member.id === thread.id)
```

Comment:
```suggestion
  const isMyThread = thread && myMembers.find((member) => member.id === thread.authorId)
```

Reply:
I think it should be something like this instead

Reply:
Oops